README.md

```diff
@@ -1,22 +1,22 @@
-# <img src="docs_app/src/assets/images/logos/Rx_Logo_S.png" alt="RxJS Logo" width="86" height="86"> RxJS: Reactive Extensions For JavaScript
+# <img src="apps/rxjs.dev/src/assets/images/logos/Rx_Logo_S.png" alt="RxJS Logo" width="86" height="86"> RxJS: Reactive Extensions For JavaScript
```

Comment:
Yeah, [it works](https://github.com/ReactiveX/rxjs/tree/b4d44efd123a845ef3770975175f6e3eb53162f3) in this case, so I guess this path will work in `CONTRIBUTING.md` file as well.